Libplanet.Benchmarks/Store.cs

```diff
@@ -0,0 +1,183 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using BenchmarkDotNet.Attributes;
+using Libplanet.Blocks;
+using Libplanet.Crypto;
+using Libplanet.Store;
+using Libplanet.Tests;
+using Libplanet.Tests.Common.Action;
+using Libplanet.Tests.Store;
+using Libplanet.Tx;
+
+namespace Libplanet.Benchmarks
+{
+    public class Store
+    {
+        private readonly ImmutableArray<Block<DumbAction>> Blocks = default;
+        private readonly int BlocksCount = default;
+        private readonly ImmutableArray<Transaction<DumbAction>> Txs = default;
+        private StoreFixture _fx = null;
+        private int TxsCount = default;
+        private IStore _store = null;
+
+        public Store()
+        {
+            var blocks = new List<Block<DumbAction>>();
+            var txs = new List<Transaction<DumbAction>>();
+            Block<DumbAction> block = TestUtils.MineGenesis<DumbAction>();
+            blocks.Add(block);
+            var key = new PrivateKey();
+            long nonce = 0;
+            for (int i = 0; i < 500; i++)
+            {
+                var blockTxs = new List<Transaction<DumbAction>>();
+                for (int j = 0; j < i % 5; j++)
+                {
+                    blockTxs.Add(Transaction<DumbAction>.Create(nonce++, key, new DumbAction[0]));
+                }
+                block = TestUtils.MineNext(block, blockTxs);
+                blocks.Add(block);
+                txs.AddRange(blockTxs);
+            }
+            Blocks = blocks.ToImmutableArray();
+            BlocksCount = Blocks.Length;
+            Txs = txs.ToImmutableArray();
+            TxsCount = Txs.Length;
+        }
+
+        [IterationSetup]
+        public void InitializeFixture()
+        {
+            _fx = new LiteDBStoreFixture();
+            _store = _fx.Store;
+        }
+
+        [IterationCleanup]
+        public void FinalizeFixture()
+        {
+            if (_store is IDisposable disposableStore)
+            {
+                disposableStore.Dispose();
```

Comment:
I guess it's okay that we only dispose of `_fx` because `LiteDBStoreFixture` has a responsibility to disposing of its own resources include `_fx.Store`.